app/.env

```diff
@@ -0,0 +1,5 @@
+PROXY_PATH=http://evbyminsd6293.minsk.epam.com:8080/
```

Comment:
Can't see where this config is used. Don't we need introduce smth like `dotenv` to have effect?

Reply:
guys, please, no any EPAM server names in config files